region/lowernorfair/east.json

```diff
@@ -842,7 +855,14 @@
                     {"or": [
                       {"and": [
                         "heatProof",
-                        "canKillYellowSpacePirates"
+                        {"enemyKill":{
+                          "enemies": [
+                            [
+                              "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)", "Yellow Space Pirate (wall)"
```

Comment:
It's ridiculously hard to catch all 3 of them for 3 double-hit PBs without taking damage. Maybe we should split them into 2 groups?

Reply:
I think splitting them into two groups is the best choice